### ISSUE TITLE
HTML entities in form submission data

### ISSUE THREAD

**Issue by anita-chouhan** [verbb/formie#1488]
### Describe the bug

We have used a Formie form to capture data from users. The submitted data is integrated with a section to create entries. If a special character (such as _&_) is submitted in form data it shows as an HTML entity in the integrated entry field.

### Steps to reproduce

1. Add a *Formie* form with a *Multi-line Text* field on the website.
2. Integrate the form submission to create entries in a section.
3. Map a *Plain Text* field of entry to form *Multi-line Text* field.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Integrations: Entries
  - Overwrite Content: Off
  - Create a New Draft: On
  - Update Entries: Off

### Craft CMS version

3.8.14

### Plugin version

1.6.27

### Multi-site?

No

### Additional context

**Sample text for submission**
```
B&B: Bed & Breakfast
R&B: Rhythm & Blues
```

**Generated data in entry field**
```
B&amp;B: Bed &amp;Breakfast
R&amp;B: Rhythm &amp; Blues
```

**Comment by engram-design** [comment]
Back-tracked this from Formie 2.x. To get this early, run `composer require verbb/formie:"dev-craft-3 as 1.6.28"`.

**Comment by anita-chouhan** [comment]
@engram-design Thank you for looking into it and providing an early fix.

**Comment by anita-chouhan** [comment]
@engram-design I updated the plugin as mentioned and it fixed the _&amp;_ issue but I noticed that the single quotes are still erratic.

For example, on submitting
`It'd be great to work with you.`

Generated data in entry field
`It&#039;d be great to work with you.`

**Comment by engram-design** [comment]
Good call, and fixed. Run `composer require verbb/formie:"dev-craft-3 as 1.6.28"` once more for the fix.

**Comment by anita-chouhan** [comment]
Thank you for the fix. It's now working for single quotes as well.

**Comment by engram-design** [comment]
Fixed in [1.6.29](https://github.com/verbb/formie/releases/tag/1.6.29)
